pkgs/swift2objc/lib/src/config.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:path/path.dart' as path;
+
+const defaultTempDirPrefix = 'swift2objc_temp_';
+const symbolgraphFileSuffix = '.symbols.json';
+const defaultModuleName = 'symbolgraph_module';
+
+class Command {
+  final String executable;
+  final List<String> args;
+
+  Command({
+    required this.executable,
+    required this.args,
+  });
+}
+
+/// Used to configure Swift2ObjC wrapper generation.
+/// See `FilesInputConfig` and `ModuleInputConfig`;
+sealed class Config {
```

Comment:
Can you make a separate `InputConfig` that `FilesInputConfig` and `ModuleInputConfig` implement, and make it a field of `Config`, rather than having `FilesInputConfig` and `ModuleInputConfig` extend `Config`. If this package is anything like ffigen, you're going to end up with dozens of fields in this `Config` class eventually, and duplicating all of them throughout the constructors of `FilesInputConfig` and `ModuleInputConfig` is quite a lot of unnecessary boilerplate.

Reply:
done